pkgs/shells/zsh/zsh-you-should-use/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-you-should-use";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "MichaelAquilina";
+    repo = pname;
+    rev = version;
+    sha256 = "0fig5ralagi5jajk7gdm52jvwql17qk9cd6j98qsndvckb26a753";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    install -D you-should-use.plugin.zsh $out/share/zsh/plugins/you-should-use/you-should-use.plugin.zsh
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/MichaelAquilina/zsh-you-should-use;
```

Comment:
Minor cosmetic issue: can you please quote this?

Reply:
Have we reached some consensus which syntax should be preferred? The majority of all packages still uses this syntax.

Reply:
Nothing official yet, it's up to you.

Reply:
Yeah, I'd stick to the current convention for now. Regardless of what comes out in the end, this kind of change can be automated.